README.md

```diff
@@ -189,6 +204,21 @@ moduleGraphConfig {
     //     )
     // ),
     // )
+
+    // You can add additional graphs.
```

Comment:
I think this is fine for now, but I think we need a cookbook page or something.
The docs are getting out of hand with too many options. 

I'll take a look at that later.